### PR TITLE
MINOR: Upgrades webpack-dev-server to use later node-forge for security reasons

### PR DESCRIPTION
**Pull request by FraukeF**
Signed-off-by: Frauke Fritz <frauke.fritz@here.com>

Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

# CI Check

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -640,6 +640,11 @@
   resolved "https://registry.yarnpkg.com/@types/node/-/node-13.13.15.tgz#fe1cc3aa465a3ea6858b793fd380b66c39919766"
   integrity sha512-kwbcs0jySLxzLsa2nWUAGOd/s21WU1jebrEdtzhsj1D4Yps1EOuyI1Qcu+FD56dL7NRNIJtDDjcqIG22NwkgLw==
 
+"@types/node@^14.0.5":
```
Comment (ninok) [inline yarn.lock:643]: Now we have two versions. Not sure why this happens though b/c `^14.0.5` also contains the above `14.11.2` version

File: yarn.lock [hunk 2]
```diff
@@ -1064,6 +1069,16 @@ ajv@*, ajv@^6.1.0, ajv@^6.10.0, ajv@^6.10.2, ajv@^6.12.2, ajv@^6.12.3:
     json-schema-traverse "^0.4.1"
     uri-js "^4.2.2"
 
+ajv@^6.1.0:
+  version "6.12.5"
```
Comment (ninok) [inline yarn.lock:1073]: same here

File: yarn.lock (hunk 1 above)
Comment (FraukeF) [reply yarn.lock:643]: as I read it, we had two versions before, they are just listed differently now

File: yarn.lock (hunk 2 above)
Comment (FraukeF) [reply yarn.lock:1073]: seems to have been a bug in yarn upgrade <pacake> for ever https://github.com/yarnpkg/yarn/issues/3967

Comment (FraukeF) [reply yarn.lock:1058]: yarn-deduplicate solves it :)

**Review by ninok — APPROVED** [review]

**Pull request merged**